### PR TITLE
meta-mender-sunxi: Added warrior to LAYERSERIES_COMPAT

### PR DESCRIPTION
**Pull request by nandra**
Verified on orangepi zero board boots fine.

Signed-off-by: Marek Belisko <marek.belisko@open-nandra.com>

### CODE REVIEW

**Comment by mirzak** [comment]
Looks good @nandra .

Though while you are at it, could you update the [README.md](https://github.com/mendersoftware/meta-mender-community/blob/sumo/meta-mender-sunxi/README.md) to the following format example in [meta-mender-odroid](https://github.com/mendersoftware/meta-mender-community/blob/warrior/meta-mender-odroid/README.md)

**Comment by nandra** [comment]
@mirzak updated. Thanks.

File: meta-mender-sunxi/README.md [hunk 1]
```diff
@@ -1,29 +1,60 @@
-# Mender integration for SUNXI boards
+# meta-mender-sunxi
 
-Supported boards:
+Mender integration layer for Odroid family of boards.
```
Comment (mirzak) [inline meta-mender-sunxi/README.md]: `Odroid -> Sunxi`

File: meta-mender-sunxi/README.md [hunk 2]
```diff
@@ -1,29 +1,60 @@
...
+
 
-Setup environment
+## Maintainer
 
-    $ . setup-environment sunxi
+The authors and maintainers of this layer are:
 
-Build
+- Mirza Krak - <mirza.krak@northern.tech> - [mirzak](https://github.com/mirzak)
```
Comment (mirzak) [inline meta-mender-sunxi/README.md]: Can you add your self here as well?

File: meta-mender-sunxi/README.md (hunk 1 above)
Comment (nandra) [reply meta-mender-sunxi/README.md]: Fixed. Force pushed.

**Comment by mirzak** [comment]
Noticed to late, but this needs to be updated,

https://github.com/mendersoftware/meta-mender-community/blob/warrior/meta-mender-sunxi/scripts/manifest-sunxi.xml

It is still pointing to `thud` on `warrior` branch

**Comment by nandra** [comment]
OK will create new PR

**Pull request merged**